src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/UserPersonDetail.kt

```diff
@@ -195,6 +236,20 @@ fun UserPersonDetail.toUserSummaryWithEmail() = UserSummaryWithEmail(
   email = staff.primaryEmail()?.email,
 )
 
+fun UserPersonDetail.toDownloadUserSummaryWithEmail() = UserSummaryWithEmail(
+  username = username,
+  staffId = staff.staffId,
+  firstName = staff.firstName.capitalizeFully(),
+  lastName = staff.lastName.capitalizeFully(),
+  active = isActive(),
+  status = accountDetail?.status,
+  locked = accountDetail?.isLocked() ?: false,
+  expired = accountDetail?.isExpired() ?: false,
+  activeCaseload = null,
```

Comment:
Still need the caseload in the report

Reply:
Will need to map twice the column with update false

Reply:
Added case load id. Can you please elaborate on last comment ('update')?